docs/explanation/realms.md

```diff
@@ -0,0 +1,42 @@
+---
+id: realms
+---
+
+# Realms
+
+A realm refers to a specific instance of a smart contract that can be written in Gnolang (Gno programming language). The potentials of realms are endless - you can create virtually any application in your mind with built-in composability, transparency, and censorship resistance. Here are some ideas of what you can build with realms:
+
+* Self-custodial financial exchanges (decentralized exchanges).
+* Lending platforms with better rates.
+* Transparent insurance systems.
+* Fair and accessible voting systems.
+* Logistics and supply chain networks.
+
+
+## Packages vs Realms 
+
+#### [**Packages**](https://github.com/gnolang/gno/tree/master/examples/gno.land/p)
```

Comment:
```suggestion
#### Packages - [`examples/gno.land/p`](../../examples/gno.land/p)
```

Reply:
This kind of link won't work, because Docusaurus expects a markdown, not a full directory link.
I've changed the phrasing, though:

[824816e](https://github.com/gnolang/gno/pull/1046/commits/824816ea27b01dbffc39519c1fc6c0df80d2cc61)